core/types/libevm.go

```diff
@@ -26,3 +29,16 @@ func WithHeaderExtras(h *Header, extra *HeaderExtra) *Header {
 	extras.Header.Set(h, extra)
 	return h
 }
+
+func BlockExtras(b *Block) *BlockExtra {
```

Comment:
This fits with the `HeaderExtras` function naming, but really it's itching me a bit: it returns a single `BlockExtra`, but it's a plural `BlockExtras`... could we have it as `GetBlockExtra` instead? 🤔

Reply:
We can rename it if it bothers you, subjectively I slightly prefer the less verbose one, but I acknowledge the inconsistency.
We can also take a final pass at renaming all these once it's put together.